docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
+                            "action" : "mark healthy"
+                        }
+                ]
+            }
+            ]
+        },
+        {
+            "address": "192.168.3.137",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "192.168.3.137:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+'
+```
+
+Our configuration has gotten quite large! However, we've gained a good bit of functionality with our new additions.
+Our servers will now be pinged every five seconds. If a the health check fails three times in a row, the associated
+terminator will be marked unhealthy, which means its precedence will be set to `failed`. If subsequently the health check
+passes three times in a row, its precedence will be reset to its original value.
+
+This example uses simple port checks, but http checks are also supported. The checks are per-terminator, so if the
+network fails between `edge-router-1` and the first application endpoint, that terminator will be marked as failed.
+However, if `edge-router-2` can still reach it, then that terminator will remain in `default` or `required`, depending
+on how it's configured.
+
+At this point we have multiple ER/Ts and multiple application endpoints thereby removing all single points of failures.
+This setup should work well for applications which are horizontally scalable.
+
+#### Health Checks
+
+There are two kinds of health checks supported, port check and http checks.
+
+**Port Checks**
+
+Port checks just check if a given port is accepting connections. They don't attempt to send or receive any data. They
+support the following properties:
+
+* `address` - an IP or DNS address with port.
+    * This field is required.
+    * Example: `192.168.1.100:8080`
+    * Example: `myserver.com:8080`
+* `interval` - how often to run the health check.
+    * This field is required.
+    * Example: `5s` (5 seconds)
+    * Example: `1m` (1 minute)
+    * Example: `250ms` (250 milliseconds)
+* `timout` - the connection timeout. Uses same format as interval.
+    * This field is required.
+    * Example: `10s` (10 seconds)
+* `actions` - how to react to health check result. Covered in more detail below.
+
+**HTTP Checks**
+
+HTTP Checks make a call to an HTTP endpoint. They support submitting a static body and checking the check results. They
```

Comment:
no headers can be added and the endpoint is expected to be unauthenticated it seems? probably should mention that?

Reply:
also "they"... another word to find/replace please